Application/People/Meta/Student/Service/Service/Transfer.php

```diff
@@ -63,24 +64,26 @@ public function getStudentTransferTypeById($Id)
     }
 
     /**
-     * @param TblStudent $tblStudent
-     * @param TblStudentTransferType $tblStudentTransferType
-     * @param TblCompany|null $tblCompany
-     * @param TblType|null $tblType
-     * @param TblCourse|null $tblCourse
-     * @param                        $TransferDate
-     * @param                        $Remark
-     * @param TblCompany|null $tblStateCompany
+     * @param TblStudent                          $tblStudent
+     * @param TblStudentTransferType              $tblStudentTransferType
+     * @param TblCompany|null                     $tblCompany
+     * @param TblType|null                        $tblType
+     * @param TblCourse|null                      $tblCourse
+     * @param string                              $TransferDate
+     * @param string                              $Remark
+     * @param TblCompany|null                     $tblStateCompany
+     * @param TblStudentSchoolEnrollmentType|null $tblStudentSchoolEnrollmentType
      */
     public function insertStudentTransfer(
         TblStudent $tblStudent,
         TblStudentTransferType $tblStudentTransferType,
         TblCompany $tblCompany = null,
         TblType $tblType = null,
         TblCourse $tblCourse = null,
-        $TransferDate,
-        $Remark,
-        TblCompany $tblStateCompany = null
+        $TransferDate = '',
+        $Remark = '',
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$Remark" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)